tests/test_spotify.py

```diff
@@ -185,5 +185,44 @@ def test_that_song_function_returns_None_when_error(self):
         self.assertRaises(SpotifyNotRunning, x)
 
 
+class ChromeTests(unittest.TestCase):
+    """
+    Unit tests for Chrome (with extension) for spotify web player
+    """
+
+    def setup(self):
+        pass
+
+    @patch('SwSpotify.spotify.get_info_chrome')
+    def test_that_artist_function_calls_get_info(self, mock):
+        """
+        test that test artist function calls get_info_chrome function
+        """
+        x = artist()
+        self.assertTrue(mock.called)
+
+    @patch('SwSpotify.spotify.get_info_chrome')
+    def test_that_song_function_calls_get_info(self, mock):
+        """
+        test that test song function calls get_info_chrome function
+        """
+        x = song()
+        self.assertTrue(mock.called)
+
+    def test_that_artist_function_returns_None_when_error(self):
+        """
+        test that test artist function returns None when the get_info_chrome function will return an error
+        """
+        x = artist
+        self.assertRaises(SpotifyNotRunning, x)
+
+    def test_that_song_function_returns_None_when_error(self):
+        """
+        test that test song function returns None when the get_info_chrome function will return an error
+        """
+        x = song
+        self.assertRaises(SpotifyNotRunning, x)
+
+
```

Comment:
Sorry I'm just noticing this but you should have tests that actually test song artist info coming up, as in `self.assertEqual(get_info_web(), (song, artist))`

Reply:
Currently it doesn't look if something worth testing is being tested

Reply:
Sure, thanks for this advice as I was not sure of the required scope for the tests. First time doing unit tests :smile:

Reply:
No problem, feel free to ask if you need any help. Unit testing is a useful skill to have :)